test/test-lib/init_mocks.ts

```diff
@@ -1,7 +1,51 @@
-import './aws-mock';
+import { app } from '../../init';
+import * as balenaToken from './balena-token';
+import * as fixtures from './fixtures';
+import { supertest, UserObjectParam } from './supertest';
 
-// override the interval used to emit the queue stats event...
-import { DeviceOnlineStateManager } from '../../src/lib/device-online-state';
-(DeviceOnlineStateManager as any)['QUEUE_STATS_INTERVAL_MSEC'] = 1000;
+export const preInit = async () => {
+	await import('./aws-mock');
 
-export {};
+	// override the interval used to emit the queue stats event...
+	const { DeviceOnlineStateManager } = await import(
+		'../../src/lib/device-online-state'
+	);
+	(DeviceOnlineStateManager as any)['QUEUE_STATS_INTERVAL_MSEC'] = 1000;
+};
+
+const getAdminUser = async () => {
```

Comment:
I moved this to be private in here as it should always be fetched via the fixtures - there's no reason to call it directly anymore